src/GeositeFramework/js/Export.js

```diff
@@ -54,63 +68,79 @@ require(['use!Geosite'],
-
-                            if ($("[name='export-include-legend']").is(":checked")) {
-                                // show & expand all legend items
-                                $("#legend-container-0").css({ visibility: "visible" });
-                                _.each(
-                                    $(".item.expand>.expand-legend"),
-                                    function (el) {
-                                        el.click();
-                                    });
-                                $(".item.extra.collapse").hide();
-
-                                // wrap all items in a div to style separately from header
-                            } else {
-                                // if the style rule is changed via jquery, that state seems to
-                                // "stick", regardless of what's in the stylesheet
-                                $("#legend-container-0").css({ visibility: "hidden" });
-                            }
-                            var pageOrientation = $("[name='export-orientation']").filter(function () { return this.checked; }).map(function () { return this.value; }).first()[0];
-                            var orientDeferred = $.Deferred();
 
+                            var pageOrientation = _.head($("[name='export-orientation']")
+                                .filter(function () { return this.checked; })
+                                .map(function () { return this.value; })
+                                .first());
                             if (pageOrientation === "Landscape") {
                                 $('<link>', {
                                     rel: 'stylesheet',
                                     href: 'css/print-landscape.css',
                                     'class': '.print-orientation-css',
                                 }).appendTo('head');
-                                _.delay(orientDeferred.resolve, 750);
-                                $(".legend-layer").each(function(el) { $(this).children(".item").wrapAll("<div class='print-legend-coll'></div>"); });
+                                _.delay(orientDeferred.resolve, 500);
                             } else {
                                 $('<link>', {
                                     rel: 'stylesheet',
                                     href: 'css/print-portrait.css',
                                     'class': '.print-orientation-css',
                                 }).appendTo('head');
-                                _.delay(orientDeferred.resolve, 750);
+                                _.delay(orientDeferred.resolve, 500);
                             }
 
                             $.when(orientDeferred).then(function () {
-                                window.print();
-                                previewDeferred.resolve();
+                                map.width = parseFloat($("#export-print-preview-map").css("width"));
```

Comment:
Instead of re-selecting `$("#export-print-preview-map")` throughout this code, save the reference to a variable which can be re-used without further DOM querying.  I think it'll also help out in readability as well.